### PR TITLE
use uncompressed assets on dev env

### PR DESCRIPTION
**Pull request by 1allen**
I'm not sure, probably `YII_DEBUG` is more appropriate there.

### CODE REVIEW

**Comment by trntv** [comment]
Yes ``YII_DEBUG`` is more appropriate here. Change PR and i'll merge it.

**Comment by 1allen** [comment]
done

File: src/widget/UploadAsset.php [hunk 1]
```diff
@@ -10,11 +10,11 @@
 class UploadAsset extends AssetBundle
 {
     public $css = [
-        'css/upload-kit.min.css'
+        YII_DEBUG ? 'css/upload-kit.css' : 'css/upload-kit.min.css'
```
Comment (trntv) [inline src/widget/UploadAsset.php]: I wouldn't work in PHP 5.5 and below. You should set these properties in ``init`` method

**Comment by 1allen** [comment]
sorry, I've distracted a bit :)

done

**Pull request merged**